sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md

```diff
@@ -20,7 +20,8 @@ irregular-plural-words:
   quota: quota
 
 rename-mapping:
-  LoadTestResource: LoadTesting
+  Resource: LoadTestingBaseResource
+  LoadTestResource: LoadTestingResource
   QuotaResource: LoadTestingQuota
   CheckQuotaAvailabilityResponse: LoadTestingQuotaAvailabilityResponse
   EncryptionProperties: CustomerManagedKeyEncryptionProperties
```

Comment:
QuotaBucketRequest: LoadTestingQuotaBucket?

Reply:
cThis is a common naming pattern for a model which is used as a method parameter, end with `Content` suffix.